benchexec/tools/cseq.py

```diff
@@ -36,13 +36,14 @@ def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
                         for example: time-limit, soft-time-limit, hard-time-limit, memory-limit, cpu-core-limit.
                         All entries in rlimits are optional, so check for existence before usage!
         """
+        tasks = list(task.input_files_or_identifier)
         assert len(tasks) == 1, "only one inputfile supported"
         inputfile = ["--input", tasks[0]]
-        spec = ["--spec", propertyfile] if propertyfile is not None else []
+        spec = ["--spec", task.property_file] if task.property_file is not None else []
         return [executable] + options + spec + inputfile
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
-        output = "\n".join(output)
+    def determine_result(self, run):
+        output = "\n".join(run.output)
```

Comment:
Please use `run.output.text` instead.

However, note that this produces different behavior than the original code: Because previously `output` contained lines with line separators, the resulting joined text would actually contain doubled line-separators (i.e., a blank line between all original lines). It is unlikely that this was intended and so the new code would actually be better, but it should be kept in mind.

Also, can the usage of `run.output.text` maybe be avoided completely? The class `RunOutput` is written specifically to provide utilities that are nice to use. For example, `if "FALSE" in output:` in this method is equivalent to `if run.output.any_line_contains("FALSE"):`.

Reply:
fixed in f86559fb27bf6844cf5515dba93b021247b2aeeb  for cseq and predatorHP.